dora/tests/integration/src/test/java/alluxio/client/fs/io/FileOutStreamIntegrationTest.java

```diff
@@ -161,7 +179,8 @@ public void writeTwoByteArrays() throws Exception {
    */
   @Test
   public void longWrite() throws Exception {
-    AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
+    String uniqPath = PathUtils.uniqPath();
```

Comment:
`PathUtils.uniqPath()` generates a path with directories. Can you just use something easy like `new AlluxioURI(mRoot).join("longWrite");`?